From 0dd2644cb5504b4798eb6fe0ba0c318c4bc728af Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Thu, 28 Aug 2014 14:21:55 -0400
Subject: [cpufreq] acpi-cpufreq: Use cpufreq_freq_attr_rw to define the cpb attribute

Message-id: <1409235879-1283-22-git-send-email-prarit@redhat.com>
Patchwork-id: 89446
O-Subject: [RHEL7.1 PATCH BZ 1134639 021/185] acpi-cpufreq: Use cpufreq_freq_attr_rw to define the cpb attribute
Bugzilla: 1134369
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Mark Langsdorf <mlangsdo@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1134639

commit 59027d356666a2c1d576baacb98de9fb30e4c48e
Author: Lan Tianyu <tianyu.lan@intel.com>
Date:   Tue Aug 13 10:05:53 2013 +0800

    acpi-cpufreq: Use cpufreq_freq_attr_rw to define the cpb attribute

    Standardise the defintion of the cpb (Core Performance Boost)
    attribute in the acpi-cpufreq driver via the cpufreq_freq_attr_rw
    macro.

    [rjw: Subject and changelog]
    Signed-off-by: Lan Tianyu <tianyu.lan@intel.com>
    Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>

Cc: Lenny Szubowicz <lszubowi@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/cpufreq/acpi-cpufreq.c b/drivers/cpufreq/acpi-cpufreq.c
index 7ecef69..8b0c7af 100644
--- a/drivers/cpufreq/acpi-cpufreq.c
+++ b/drivers/cpufreq/acpi-cpufreq.c
@@ -197,7 +197,7 @@ static ssize_t show_cpb(struct cpufreq_policy *policy, char *buf)
  return sprintf(buf, "%u\n", boost_enabled);
 }
 
-static struct freq_attr cpb = __ATTR(cpb, 0644, show_cpb, store_cpb);
+cpufreq_freq_attr_rw(cpb);
 #endif
 
 static int check_est_cpu(unsigned int cpuid)
-- 
1.7.1